crypto/keys/output.go

```diff
@@ -7,19 +7,29 @@ import (
 // KeyOutput defines a structure wrapping around an Info object used for output
 // functionality.
 type KeyOutput struct {
-	Name      string                 `json:"name"`
-	Type      string                 `json:"type"`
-	Address   string                 `json:"address"`
-	PubKey    string                 `json:"pubkey"`
-	Mnemonic  string                 `json:"mnemonic,omitempty"`
-	Threshold uint                   `json:"threshold,omitempty"`
-	PubKeys   []multisigPubKeyOutput `json:"pubkeys,omitempty"`
+	Name      string                 `json:"name" yaml:"name"`
+	Type      string                 `json:"type" yaml:"type"`
+	Address   string                 `json:"address" yaml:"address"`
+	PubKey    string                 `json:"pubkey" yaml:"pubkey"`
+	Mnemonic  string                 `json:"mnemonic,omitempty" yaml:"mnemonic"`
+	Threshold int                    `json:"threshold,omitempty" yaml:"threshold"`
+	PubKeys   []multisigPubKeyOutput `json:"pubkeys,omitempty" yaml:"pubkeys"`
+}
+
+// NewKeyOutput creates a default KeyOutput instance without Mnemonic, Threshold and PubKeys
+func NewKeyOutput(name, keyType, address, pubkey string) KeyOutput {
+	return KeyOutput{
+		Name:    name,
+		Type:    keyType,
+		Address: address,
+		PubKey:  pubkey,
+	}
 }
 
 type multisigPubKeyOutput struct {
-	Address string `json:"address"`
-	PubKey  string `json:"pubkey"`
-	Weight  uint   `json:"weight"`
+	Address string `json:"address" yaml:"address"`
+	PubKey  string `json:"pubkey" yaml:"pubkey"`
+	Weight  int    `json:"weight" yaml:"weight"`
```

Comment:
I would revert this. What was the rationale for changing this?

Reply:
Otherwise it prints “0x...”

Reply:
Where does this get printed? I don't recall this happening before? It should should print the weight/number

Reply:
On the tests I wrote. Also this is only the format for the CLI output

Reply:
^^ @alexanderbez

Reply:
I don't follow @fedekunze...Which test? `Weight` is already a `uint` so why cast it to an `int`? What does this have to do with printing `0x..`?

Reply:
You are right. With the output. Sorry, my bad. Reverted